ui/number-field.reel/number-field.js

```diff
@@ -1,40 +1,50 @@
 /**
-    @module "montage/ui/native/input-number.reel"
-    @requires montage/ui/component
+    @module "montage/ui/number-field.reel"
     @requires montage/ui/text-input
 */
 
-var TextInput = require("ui/text-input").TextInput;
+var TextInput = require("ui/text-input").TextInput,
+    MinMaxConverter = require("core/converter/min-max-converter.js").MinMaxConverter;
+    
 
 /**
- * Wraps the a &lt;input type="date"> element with binding support for the element's standard attributes.
-   @class module:"montage/ui/native/input-number.reel".InputNumber
+ * Wraps the a &lt;input type="number"> element with binding support for the element's standard attributes.
+   @class module:"montage/ui/number-field.reel".NumberField
    @extends module:montage/ui/text-input.TextInput
  */
 var NumberField = exports.NumberField = TextInput.specialize({
-	hasTemplate: {value: false }
+
+    enterDocument:{
+        value: function() {
+            this.converter = new MinMaxConverter();
```

Comment:
RangeEnforcer, we probably don't need to add -Infinity nor Infinity for it to work after all